.github/workflows/medicines-web-branch.yaml

```diff
@@ -48,6 +48,16 @@ jobs:
         working-directory: medicines/web
         run: yarn build && yarn export
 
+      - name: Write variables to .env
+        working-directory: medicines/web
+        run: |
+          {
+            echo "AZURE_SEARCH_SERVICE=\"$AZURE_SEARCH_SERVICE\""
+            echo "AZURE_SEARCH_INDEX=\"$AZURE_SEARCH_INDEX\""
+            echo "AZURE_SEARCH_KEY=\"$AZURE_SEARCH_KEY\""
+            echo "GRAPHQL_URL=\"$GRAPHQL_URL\""
+          } > .env
```

Comment:
is this because `yarn` doesn't read from env but from `.env`?

Reply:
I updated the `yarn test-e2e` script to read from `.env` just as a convenience for people so I had to add this so that `.env` exists when that script is called in CI